qa/tasks/ceph_manager.py

```diff
@@ -23,7 +24,8 @@
 from teuthology.orchestra.remote import Remote
 from teuthology.orchestra import run
 from teuthology.exceptions import CommandFailedError
-import six
+from tasks.thrasher import Thrasher
```

Comment:
@tchaikov I see some missing code that needs to be backported because of which I am seeing https://jenkins.ceph.com/job/ceph-api/6116/ failure should I backport 8623bd33d0f85ba529d2808cc44cc1dafd32d1b1 to this PR itself?